proceeding/tables/catalog_table.tex

```diff
@@ -0,0 +1,21 @@
+% Table
+\begin{table}[h]
+
+\caption{Catalog information.}
+\label{tab:a}
+\tabcolsep7pt\begin{tabular}{ || lrll ||}
+\hline
+% \tch{1}{c}{b}{Catalog\\ name} & \tch{1}{c}{b}{Number\\ of sources}  & \tch{1}{c}{b}{Update}  & \tch{1}{c|}{b}{Description}  \\
+% & \tch{1}{c}{b}{Single\\ outlet}  & \tch{1}{c}{b}{Small\\ multiple\tabnoteref{t1n1}}  & \tch{1}{c}{b}{Large\\ multiple}  & \tch{1}{c}{b}{Total}   \\
+
+
+\textbf{Catalog} & \textbf{Sources} & \textbf{Update} & \textbf{Description} \\ \hline
```

Comment:
Remove all the manual `\textbf`?
(I think it's uncommon, plus does it really help with readability?)